ComfySoda_Unity2018.2.7f1/Assets/Scripts/ComySodaEngine/EnemyLogic.cs

```diff
@@ -21,21 +21,33 @@ public class EnemyLogic : MonoBehaviour
         public float enemysCurrentMotivation = 100.0f;
         float enemyDefaultMotivation;
         enum EnemyState { GO_TO_NEAREST_WAYPOINT, BEE_LINE_FOR_PREY, PHASE_TO_WAYPOINT, FADE_IN, FOLLOW_WAYPOINTS };
+        [SerializeField]
         EnemyState currentEnemyState = EnemyState.FADE_IN;
+        [SerializeField]
         EnemyState previousEnemyState = EnemyState.FADE_IN;
+        [SerializeField]
         Waypoint lastTriggeredWaypoint = null;
+        [SerializeField]
         Waypoint nextTargetedWaypoint = null;
         Rigidbody2D myRigidbody2D;
         Animator tempAnimator;

```

Comment:
Why is the Animator temporary?

Reply:
because at one point it was temporary and I never renamed it.  It's fixed now.  Thanks.